templates/settings.py.erb

```diff
@@ -1,3 +1,11 @@
+########################################################################################
+# WARNING: This file is externally managed and you should never make manual edits.     #
+#                                                                                      #
+# Not only are your edits likely to be overwritten, there is also a strong possibility #
+# of breaking your system. Refer to documentation for the software that installed      #
+# Pulpcore to determine the safe and persistent way to modify the configuration.       #
+########################################################################################
```

Comment:
If we are going to solve this problem *now* (as this is a change that is nice but not needed by the target of this PR and could be addressed later) could we stick to the Foreman header for consistency? https://github.com/theforeman/puppet-foreman/blob/master/templates/_header.erb

Reply:
I think it would make a good separate PR. There's no reason it really needs to be here in this PR.

Reply:
I am wary of removing the unit file for pulpcore-resource-manager.service without including this warning in settings.py

Reply:
Specifically that change invites the possibility of someone modifying the `USE_NEW_WORKER_TYPE` setting without having the necessary service definitions in place

Reply:
Regarding the existing template which mentions that the file is managed by a puppet module, I am also concerned this is not clear to foreman-installer users, or downstream users.

Reply:
Thing is, this is a problem at that already exists. So we should fix it separately and consider consistency. We run the risk of bike shedding this PR and getting behind schedule. I think it will be more productive to focus this PR on the actual change, and then make the proposed change here separately. You are also banking on users actually *reading* as well ;)

Reply:
My main motivation is that we can then merge this PR without bikeshedding on the exact message. It's also nice in the changelog to have a dedicated item.

Reply:
Opened https://github.com/theforeman/puppet-pulpcore/pull/210